src/bionics.cpp

```diff
@@ -85,20 +203,26 @@ bionic_data const& bionic_info(std::string const &id)
 
     debugmsg("bad bionic id");
 
-    static bionic_data const null_value {"bad bionic", false, false, 0, 0, 0, 0, 0, "bad_bionic", false};
+    static bionic_data const null_value {"bad bionic", false, false, 0, 0, 0, 0, 0,
+                                         "bad_bionic", false,
+                                         { num_bp , 1 } };
```

Comment:
I need some help here. This line causes failure but I have no idea how to do this right (later it will be vector of maps but currently I have an issue even with simple map).